docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,323 @@ redirect_from:
+
+This page contains an overview of your app in addition to important credentials you'll need later, like the `Signing Secret` under the **App Credentials** header. 
+
+![Basic Information page](../assets/basic-information-page.png "Basic Information page")
+
+Look around, add an app icon and description, and then let's start configuring your app 🔩
+
+---
+
+### Tokens and installing apps
+Slack apps use [OAuth to manage access to Slack's APIs](https://api.slack.com/docs/oauth). When an app is installed, you'll receive a token that the app can use to call API methods. 
+
+There are two token types available to a Slack app: user (`xoxp`) and bot (`xoxb`) tokens. User tokens allow you to call API methods on behalf of users after they install or authenticate the app. There may be several user tokens for a single workspace. Bot tokens are associated with bot users, and are only granted once in a workspace where someone installs the app. The bot token your app uses will be the same no matter which user performed the installation. Bot tokens are the token type that _most_ apps use.
+
+For brevity, we're going to use bot tokens for this guide.
+
+Navigate to the **OAuth & Permissions** on the left sidebar and scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**.
+
+For now, we'll just add one scope: [`chat:write`](https://api.slack.com/scopes/chat:write). This grants your app the permission to post messages in channels it's a member of.
+
+Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**. You'll be led through Slack's OAuth UI, where you should allow your app to be installed to your development workspace.
+
+Once you authorize the installation, you'll land on the **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
+
+![OAuth Tokens](../assets/bot-token.png "Bot OAuth Token")
+
+> 💡 Treat your token like a password and [keep it safe](https://api.slack.com/docs/oauth-safety). Your app uses it to post and retrieve information from Slack workspaces.
+
+---
+
+### Setting up your local project
+With the initial configuration handled, it's time to set up a new Bolt project. This is where you'll write the code that handles the logic for your app.
+
+If you don’t already have a project, let’s create a new one. Create an empty directory:
+
+```shell
+mkdir first-bolt-app
+cd first-bolt-app
+```
+
+<!-- TODO - Do we want to explain how to install the `pyenv` runtime management in a <details> section? -->
+Next, we recommend using a [Python virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) to manage your project's dependencies. This is a great way to prevent conflicts with your system's Python packages. Let's create and activate a new virtual environment:
```

Comment:
`slack_bolt` requires Python 3.6 or later. Version 3.6 was released in December 2016, it's almost 4 years old.

**Should we add a `<details>` section to help developers install a compatible version of Python?** In the section, we could help developers [install `pyenv` for runtime management](https://github.com/slackapi/bolt-python/blob/main/.github/maintainers_guide.md#python-and-friends) and install Python 3.8.

Reply:
Python set up is tricky, and I think the suggestion to help developers install a compatible version is a good one! That being said, I don't know if aligning with pyenv is the best option. First of all, it doesn't directly support Windows, which I believe is much more important to the average Python developer. I've also had mixed experiences getting it to play well with the system Python on macOS - but that was early days maybe the project is well past this now.

What are the disadvantages in comparison to linking to [Python's official download page](https://www.python.org/downloads/)? It does some user agent sniffing to show the visitor an installer for their own operating system. I would expect the PSF to maintain this page with the best general installation option for most users, even if their recommendation changes over time.

Reply:
I would just note that it requires Python 3.6 or later and link out to the Python website, unless you think a significant amount of devs might get tripped up by it.

Reply:
Thanks for the discussion @aoberoi and @shaydewael. I'll keep it simple and focused by only mentioning that we require Python 3.6 or later (linking to the official download page). Fixed in commit fbea9ec

Preview:
![image](https://user-images.githubusercontent.com/21328/93270522-2eac7380-f766-11ea-89b4-f517ba2a3162.png)